scripts/cast-on-tenderly/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "deploy-to-tenderly",
+  "type": "module",
+  "version": "1.0.0",
+  "description": "A script to execute spell on a fork",
+  "main": "index.js",
+  "scripts": {
+    "do": "node ."
```

Comment:
Why not use the traditional `start` script here?

Reply:
changed to `start`!